device_kylepro.mk

```diff
@@ -41,6 +41,14 @@ PRODUCT_PACKAGES += \
     e2fsck \
     setup_fs
 
+# F2FS tools
+ifeq ($(BUILD_TWRP), true)
+PRODUCT_PACKAGES += \
+    mkfs.f2fs \
+    fsck.f2fs \
+    fibmap.f2fs
+endif
+
```

Comment:
no, this is platform tools, why put it there if you're planning to build TWRP? TWRP have its own sets of tools